src/meltano/core/tracking/iglu-client-embedded/schemas/com.meltano/environment_context/jsonschema/1-1-0

```diff
@@ -0,0 +1,180 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a Snowplow context describing the environment in which the Meltano process is executing",
+    "self": {
+        "vendor": "com.meltano",
+        "name": "environment_context",
+        "format": "jsonschema",
+        "version": "1-1-0"
+    },
+    "type": "object",
+    "properties": {
+        "context_uuid": {
+            "description": "A UUID that uniquely identifies an instance of this context",
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+            "minLength": 36,
+            "maxLength": 36
+        },
+        "parent_context_uuid": {
```

Comment:
Sharing the main diff of this file vs the 1-0-0 version.

<img width="1314" alt="Screen Shot 2022-11-30 at 9 17 22 AM" src="https://user-images.githubusercontent.com/27376735/204819413-64bc7240-eed3-4950-9c66-9f48ba8760c2.png">